### DISCUSSION TITLE
Increment a field (int) in Supabase-js - React Native Expo

### DISCUSSION THREAD

**Discussion by Atharva2628** [supabase/supabase#1961] [Questions] [answered]
_Hi everyone,
(prolly a very stupid question)
But I wasn't able to find the answer anywhere including the doc and this is my first time asking here._

I'm looking for something along the lines of:
`let { data, error } = await supabase.from("Table").update({ Field1: ++ }).match({ id: 1 });`
Increment `+1` to the value of `Field1` from the table `Table` at row id `1`

_The project is a react native expo project if that helps_

**Comment by inian** [comment, marked as answer]
Check out the previous discussion on this @Atharva2628 https://github.com/supabase/supabase/discussions/909#discussioncomment-516407

The function is also part of our quickstart guide - /project/your-project-ref/editor/sql

**Reply by Atharva2628** [reply]
Thanks mate 😃
